app/models/concerns/facets/managed_host_extensions.rb

```diff
@@ -0,0 +1,66 @@
+require 'facets'
+
+module Facets
+  module ManagedHostExtensions
+    extend ActiveSupport::Concern
+
+    included do
+      Facets::ManagedHostExtensions.refresh_facet_relations(self)
+
+      def attributes
+        hash = super
+
+        # include all facet attributes by default
+        facets_with_definitions.each do |facet, facet_definition|
+          hash["#{facet_definition.name}_attributes"] = facet.attributes.reject { |key, _| %w(created_at updated_at).include? key }
+        end
+        hash
+      end
+    end
+
+    def self.refresh_facet_relations(klass)
```

Comment:
Nitpick, it's a bit more idiomatic to use `class << self` to avoid having to call `self.method` when you define several class methods, see https://github.com/bbatsov/ruby-style-guide#def-self-class-methods